news/7402.bugfix

```diff
@@ -0,0 +1 @@
+Reject VCS URLs with an empty revision.
```

Comment:
```suggestion
Reject VCS URLs with an empty revision.

```

Reply:
This should fix the linting error.

Reply:
I already added a new line in one of the commits, and now all checks have passed.